### PR TITLE
Document changes that fixed issue #958

### PR DESCRIPTION
**Pull request by tst-rwildes**
Documents fixes for #958 

Added new optional build parameters that add the needed flexibility to the build process of Open GEE for building code from less than optimal i/o devices.

### CODE REVIEW

File: docs/geedocs/5.2.4/answer/7160004.html [hunk 1]
```diff
@@ -44,6 +44,8 @@ <h5>
   </h5>
     <p><strong>C++98 no longer supported</strong>. As of 5.2.4 C++11 syntax can be used and building open GEE using C++98 standard will no longer be supported</p>
     <p><strong>Added "Save and Build" file menu option</strong>. As of 5.2.4 the Fusion UI will include an additional file menu option that allows users to save and build assets with one click, instead of clicking the save option and then the build option immediately after.</p>
+    <p><strong>Added build parameter to redirect the build output</strong>Build output can now be optionally directed to a folder of your choosing instead of the current hard coded defaults. See earth_enterprise/BUILD.md in source code for more details.</p>
+    <p><strong>Added experimental build cache folder parameter to the build</strong>Currently this was added as an experimental option to potentially speed up builds. Some more work remains to be done before using this option in production builds. Use of this option is not recommend except for giving feedback and/or helping to address issues with open GEE's build process using scons cache. See earth_enterprise/BUILD.md in source code for more details.</p>
```
Comment (tst-jhaight) [inline docs/geedocs/5.2.4/answer/7160004.html]: Typo: `not recommend` -> `not recommended`

File: earth_enterprise/BUILD.md [hunk 2]
```diff
@@ -61,15 +69,16 @@ platforms on how to setup the dependencies, tools, and compilers.
         the configuration to run again, otherwise the scons build may complain
         about missing libraries
 
-1. Build Earth Enterprise Fusion and Server:
+2. Build Earth Enterprise Fusion and Server:
```
Comment (tst-jhaight) [inline earth_enterprise/BUILD.md]: I'm not positive this is needed but it also doesn't appear to hurt (see line 45 above and view it in the preview).

Also should line 24 be a continuation of the numbered list or start over at 1?  IMO it should be a continuation but no idea how to make the markdown interpret it as such.

File: docs/geedocs/5.2.4/answer/7160004.html (hunk 1 above)
Comment (tst-blaird) [reply docs/geedocs/5.2.4/answer/7160004.html]: For both of the new items: I think after the close of the strong tag there needs to be a period and a space.

File: earth_enterprise/BUILD.md (hunk 2 above)
Comment (tst-blaird) [reply earth_enterprise/BUILD.md]: Yes, it does look like step 1 occurs three times in the above text.

File: docs/geedocs/5.2.4/answer/7160004.html [hunk 3]
```diff
@@ -44,6 +44,8 @@ <h5>
   </h5>
     <p><strong>C++98 no longer supported</strong>. As of 5.2.4 C++11 syntax can be used and building open GEE using C++98 standard will no longer be supported</p>
     <p><strong>Added "Save and Build" file menu option</strong>. As of 5.2.4 the Fusion UI will include an additional file menu option that allows users to save and build assets with one click, instead of clicking the save option and then the build option immediately after.</p>
+    <p><strong>Added build parameter to redirect the build output</strong>. Build output can now be optionally directed to a folder of your choosing instead of the current hard coded defaults. See earth_enterprise/BUILD.md in source code for more details.</p>
+    <p><strong>Added experimental build cache folder parameter to the build</strong>. Currently this was added as an experimental option to potentially speed up builds. Some more work remains to be done before using this option in production builds. Use of this option is not recommened except for giving feedback and/or helping to address issues with open GEE's build process using scons cache. See earth_enterprise/BUILD.md in source code for more details.</p>
```
Comment (tst-blaird) [inline docs/geedocs/5.2.4/answer/7160004.html]: The change to add the ". " after the closing of the strong tag looks good but I think the comment that @tst-jhaight made still needs to be addressed.  In "Use of this option is not recommened" the last word needs to be changed to "recommended".

Also, in "helping to address issues with open GEE's build process" does the word "open" need to be "Open"?

And thank you for fixing the numbering in the BUILD.md file.

**Pull request merged**